src/testRunner/unittests/tsc/runWithoutArgs.ts

```diff
@@ -4,7 +4,8 @@ namespace ts {
             scenario: "runWithoutArgs",
             subScenario: "show help with ExitStatus.DiagnosticsPresent_OutputsSkipped",
             fs: () => loadProjectFromFiles({}),
-            commandLineArgs: []
+            commandLineArgs: [],
+            environmentVariables: { TS_TEST_TERMINAL_WIDTH: "120" }

```

Comment:
Add test without setting width as well

Reply:
I don't think we can't do this, because it will be different depending on what the width of someone's terminal would be when running the tests (aka it'd be inconsistent)

Reply:
Though maybe having a smaller value would also be a good idea

Reply:
Default is suppose to be fixed if the host does not provide the method.